lib/ui/semantics/semantics_node.h

```diff
@@ -67,6 +68,14 @@ struct SemanticsNode {
   std::vector<int32_t> children;
 };
 
+/**
+ * Contains semantic nodes that need to be updated.
+ *
+ * The keys in the map are stable node IDd, and the values contain
+ * semantic information for the node corresponding to the ID.
+ */
+using SemanticsNodeUpdates = std::unordered_map<int32_t, SemanticsNode>;
```

Comment:
We don't use javadoc/headerdoc format for docstrings in the engine. Lets follow the CPP guide for consistency [go/fluttercpp#Function_Comments](go/fluttercpp#Function_Comments).

Reply:
Done.